projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/strategy/datajob/JobFieldStrategyByLastExecutionStatus.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.graphql.strategy.datajob;
+
+import com.vmware.taurus.controlplane.model.data.DataJobExecution;
+import com.vmware.taurus.service.graphql.model.Criteria;
+import com.vmware.taurus.service.graphql.model.Filter;
+import com.vmware.taurus.service.graphql.model.V2DataJob;
+import com.vmware.taurus.service.graphql.strategy.FieldStrategy;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+import org.springframework.util.CollectionUtils;
+
+import java.util.Comparator;
+import java.util.function.Predicate;
+
+@Component
+public class JobFieldStrategyByLastExecutionStatus extends FieldStrategy<V2DataJob> {
+
+   private static final Comparator<V2DataJob> COMPARATOR_DEFAULT = Comparator.comparing(
```

Comment:
Is the comparison alphabetical?

Reply:
In my opinion the API consumer will expect the response to be sorted alphabetically.

Reply:
Agreed and changed it accordingly.